test/CommoditySpec.java

```diff
@@ -0,0 +1,356 @@
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import play.libs.Json;
+import play.mvc.Result;
+import play.test.Helpers;
+import play.mvc.Http.RequestBuilder;
+import play.test.FakeApplication;
+import play.core.j.JavaResultExtractor;
+
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Before;
+import static org.junit.Assert.*;
+
+import com.fasterxml.jackson.databind.node.JsonNodeFactory;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import io.pathfinder.models.Commodity;
```

Comment:
If we put this test file in `./test/io/pathfinder/models` we don't need this import.